cobalt/media/base/sbplayer_pipeline.cc

```diff
@@ -702,6 +702,8 @@ void SbPlayerPipeline::OnDemuxerError(PipelineStatus error) {
     return;
   }
 
+  LOG(INFO) << "SbPlayerPipeline::OnDemuxerError() called with error " << error;
```

Comment:
If `error = ::media::PIPELINE_OK`, would we still log this as error?

Reply:
`OnDemuxerError()` shouldn't be called with `::media::PIPELINE_OK`, and the check below is just to be safe.